### PR TITLE
Add v1.6.0 of mattermost-plugin-zoom to the Marketplace

### PR DESCRIPTION
**Pull request by agarciamontoro**
#### Summary
Generated with `go run ./cmd/generator/ add mattermost-plugin-zoom v1.6.0 --official --on-prem`.

Changelist at https://github.com/mattermost/mattermost-plugin-zoom/releases/tag/v1.6.0.

#### Ticket Link
--

### CODE REVIEW

**Review by mickmister — APPROVED** [review]

File: plugins.json
Comment (mickmister) [inline plugins.json:10520]:
```suggestion
    "release_notes_url": "https://github.com/mattermost/mattermost-plugin-zoom/releases/tag/v1.6.0",
```

**Review by mickmister — CHANGES_REQUESTED** [review]
Not sure how these got set this way

File: plugins.json
Comment (agarciamontoro) [inline plugins.json:10650]: These are also wrong
```suggestion
    "platforms": {
      "linux-amd64": {
        "download_url": "https://plugins-store.test.mattermost.com/release/mattermost-plugin-zoom-v1.6.0-linux-amd64.tar.gz",
        "signature": "iQIzBAABCAAdFiEExViBuA9p6GO4WtXR0bVLR6XO/sQFAmFTTCYACgkQ0bVLR6XO/sSgRxAAlRfYHBqbgeBVSVuwwZooqkRPCa0VEqGRKs82iz0o5HV/BpZ3CNf0iNTJlD42ftIjiVHgB7JS4KMF+TpKqUD6GxW+fz40D0dqStyyP+YgYMP5GL23V3ml/gdEbnjBaOsT75tr4NMDAjR1cfk8QodIQMExe1Lr1eLqzK1JPZBVz1bdE2LKH7ZMSskgVdKCFYV3YaV7M0bNYMKalo/EMZ1RavtsUZPqVXHmJAzdREP/DWWaeokVenX2EHmr3zFFIaHGm4Ueq6jZFr9oEdUdK2CfhM8yo+pUth5oDuSOLoDUzcBvMKYDMhfYhWYuV/umA9wkEyJpQnlncht8BaaD5x+vUZ6VD7rupVmS5w3M4HleOXerXusT5RW8KsMmMGvNMUDgrS0qp/QtNqJDQuQ26P4qejOBF1TiBC36rBAqsJFH/aJR0Dtw9vVaW/JlkP1phF44Ya3FMVMuqTfekz2OdJKFytC3lXvPMLSdVFv7UcgHoU/J9GGjnVuzJ9jGhuda95TRb/loKSOcdwQOPmFhpNWj2YAgJxW97E73oyc/Sck2U7rhiG+n721/0iVbCHsNZCIYyTJ6BceizbeUqCoBXEkyCIIeTh74l1guBuNil+c58Kg0jXhU4bL6hqJH/JZNDQNKnCk9Ha1IT8JqxB2wbI01zxgAdgXQopi2zfTKGRBNkL0="
      },
      "darwin-amd64": {
        "download_url": "https://plugins-store.test.mattermost.com/release/mattermost-plugin-zoom-v1.6.0-osx-amd64.tar.gz",
        "signature": "iQIzBAABCAAdFiEExViBuA9p6GO4WtXR0bVLR6XO/sQFAmFTTCMACgkQ0bVLR6XO/sReVBAAgjCYGEObO3mzY9AHbo1w5qfYtSePUx3sa+iiHjBM/yWPM/lT3kBu9VZq4pw/jrOS3iOPJOw0VTEI3ENOXm+PAPsVpUMSdWnhV5/BUIs9xYqXpsRZvaCrCeM6OfY4sDGTrXiMAqSJow6EtOnmFSZXkJfQVN3x5qOJJoCdUqYVQ8+LyLWBBDcM5aiuNqbDBw+Sp+EKCCxabimOr7pUene0doFNP68GXL4WErACbrCVOpUjSjcaSXFAmH8wC+nHtW2/BFJIJAHpUeegiIjgm/RZOcB7tw04G6gYV8+nUa0A8kCQicBTnCJq3YH5tuFFyytWLlPVqceH6b9FHNjuSnVXvmcuRjM/goIM6+B4FRR7hQj8uqxWVJUnQJxKBmCu7/vAaLGQCVslfKAJstTgUJtxsEWjSt5BAQvoTX1ts7R+PAnryoCSABCquEA7wqM0PM5pPes3KJDhj9AgI+6iHfZtCyAp+g2w4ZnlknVJRFrNucMqF8zHlthUEQgKyAQ7UxfWx89QR3WkVnVwKn+ul1Mu1vbkDEwTkeNerUVEbpFRxCxzpiXqJKUX4EUJOvxGrzIaNoN1QrKbRmnqUOOS+AkestlMKd0khiObWGtMChsMtvPNy+6Ef9ywgEYWr0gJBvhEgC3YuMO8MiJIlB3wDTdXZK/NhQODsSrBleboOQ1+JBY="
      },
      "windows-amd64": {
        "download_url": "https://plugins-store.test.mattermost.com/release/mattermost-plugin-zoom-v1.6.0-windows-amd64.tar.gz",
        "signature": "iQIzBAABCAAdFiEExViBuA9p6GO4WtXR0bVLR6XO/sQFAmFTTCQACgkQ0bVLR6XO/sQUuw/+PLuGuayGdCzPEAQ7m2yrLiRxYY+YaHNuFfG/Q8XwE4gLQk5cENZ8q5dAjmcQHLRxVfmnQGRhDUgEbYI1qIXcztEvFUKAwnyzRFtlCEc88aZKnBWYf0FNNJULCs3egsoG9bndl/IfYvSqyvcYh6/DXAOgo2FG7wU2iOw+mT6SMGB5z1bm3XrtoeR6uuwKc2BYyT38Fom3rmEQjc9eemnPaQ5HZRJbjUhLVcQm114c89sBK7QUJ4MU9xcs1kcPD1l6KtgV+6NwoL+tsUCVUVgNQWfzUyLPOLaFwNUrBX/IpHCHvvq2rrSJvrEmrmdhyHBNwb16MCfA+RC7XkdsyRurhhm7vrrC/SjIEhDvvJwwHULbtv+lfp+whJLG3acizjJLzbtMua6uRo0PrBmWRoLHXT1h1h+VS7TAjAG9eRgCpl0hlw4yiwgDmgBEG/6w4huU12iJC0u6po9DW3XvcBXagFRQHboaPkZtLdHOomi/y9L/Vu6c0oZ0YMPr0HHbVFsQ4bwC1k2ExlpK6gNGis8NAkhkqMkEMbq1I5Z44BNaefZhjJWn/2C6Tf2ki+7PEEnQh4RfQGRJJELXJT1cxKApxE7V3izHa/WSehY0ibI33+vZFCvqHMeIdbBnwbm0KFfjepXTMRJxKYUlS2Cv8uMC294vNeBfrarXY261f/7qhzw="
      }
    },
```

**Comment by agarciamontoro** [comment]
@mickmister I tried re-running the command and it still got me the old 1.5.1 versions in those places, so I manually changed it. The cloud version PR seems to be ok, though https://github.com/mattermost/mattermost-marketplace/pull/287

**Comment by mickmister** [comment]
@agarciamontoro I've verified that the `download_url` in both this PR and https://github.com/mattermost/mattermost-marketplace/pull/286 contain a webapp bundle with the App Bar component, and the included `plugin.json` file says version `1.6.0` and `1.6.0-cloud` respectively. So it seems there are no issues functionally with patching this here. Still curious why the version is different here though.

File: plugins.json
Comment (larkox) [inline plugins.json:10530]: Not blocking now, but @mickmister we should probably update this description at some point, since a few lines lower it says "min_server_version: 5.12.0".

**Review by larkox — APPROVED** [review]

**Review by saturninoabril — APPROVED** [review]
LGTM, will test after merged.

**Review by mickmister — APPROVED** [review]

**Comment by mickmister** [comment]
When trying to install the Zoom plugin 1.6.0 from the marketplace, I see this error:

image

@agarciamontoro This may be an artifact of our discussion above regarding mismatching versions. Maybe something went wrong with the original bundling/signing

**Pull request merged**
